.travis.yml

```diff
@@ -102,6 +102,13 @@ script:
     if [ "$ANALYSIS" = "1" ]; then
       vendor/bin/phpstan analyze --memory-limit=256M
     fi
+  - |
+    if [ "$E2E" = "1" ]; then
+      npm run build:js
+      npm run env:start
+      npm run env:e2e:ci
+      npm run env:stop
```

Comment:
Why do we care about stopping the env?

Reply:
Don't want anything to leak and potentially cause the build to time out because the env is still running.

Reply:
Why would that happen. E2E run in it's own process.

Reply:
I'm arguing that cleaning up after these tests here finish is a good thing to do, and avoids any issues where the CI environment might not recognize tests being finished and doesn't stop containers, until it just times out doing nothing.